hydra-cluster/README.md

```diff
@@ -40,13 +40,29 @@ to the `config/credentials/faucet.sk` on that network. The Hydra nodes
 can reference pre-existing contracts living at some well-known
 transaction or can post a new transaction to use those contracts.
 
+:warning: do not provide actual funds to this faucet address as the
+signing key is publicly available. Shall you want to run the smoke
+test with actual funds, you shall override these file to use a secret
+signing key safely stored. See how the C.I overrides these files in
+.github/workflows/smoke-test.yaml#51
```

Comment:
Could: not refer to a line, the comment might go out of date quickly and unnoticed. A reference to the workflow file should suffice.

Reply:
👍